### PR TITLE
Ana/5189 add hardcoded need info for extension

### LINKED ISSUE #5189: Hardcode needed network information into the extension
**Is your feature request related to a problem? Please describe.**

If we shutdown the API the extension will not work anymore.

**Describe the solution you'd like**

Hardcode the needed (only what is needed) network infos into the extension code.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #5189 

**Description:**

Looking good

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by Bitcoinera** [comment]
Guess we should also update the README about this, so people now how to add their own data.

Should I add correct validators & networks data? Or just one as example is enough, right

**Review by faboweb — COMMENTED** [review]
Are validators also looked up from the APi?

**Comment by Bitcoinera** [comment]
> Are validators also looked up from the APi? If yes, don't resolve images and names. Just show addresses

Well, yes, they are looked up from API which looks them up from DB. But we are using names in the extension (Transaction Details). Should I change that to be always addresses? I guess so

**Comment by faboweb** [comment]
Yes, let's change that to always be addresses if the API is not available

**Comment by faboweb** [comment]
🚀 Deployed on https://5fa971aea3d863002f203a38--lunieio.netlify.app

File: extension/src/components/SessionApprove.vue [hunk 1]
```diff
@@ -90,6 +90,7 @@ import TransactionItem from 'transactions/TransactionItem'
 import TableInvoice from 'src/ActionModal/components/TableInvoice'
 import Address from 'common/Address'
 import { required } from 'vuelidate/lib/validators'
+import hardcodedValidators from '../../validators'
```
Comment (faboweb) [inline extension/src/components/SessionApprove.vue:93]: I would have prefered if you would have left this out. This wasn't necessary

**Review by faboweb — COMMENTED** [review]
Missing error handling when network is not found

**Comment by faboweb** [comment]
Doesn't work
image

**Comment by Bitcoinera** [comment]
Weird. It also failed for me, but I don't get the same error than you there. I get this on the background page:

image

**Comment by Bitcoinera** [comment]
Ah no, now it worked:

image


Double weird

**Comment by faboweb** [comment]
Akash testnet claiming: (no tx shows)
image

**Comment by faboweb** [comment]
Fixed errors. Please double check and merge

**Review by faboweb — DISMISSED** [review]

**Comment by Bitcoinera** [comment]
Double checking

**Comment by Bitcoinera** [comment]
It doesn't work :sweat: 

image

No validator displayed

**Comment by Bitcoinera** [comment]
I will fix it in a bit :v:

**Comment by Bitcoinera** [comment]
OK, no, it is all good :v:

**Pull request merged**